backend/scraper/scrape_walmart.py

```diff
@@ -1 +1,100 @@
-# TODO scrape items from Walmart
+from selenium import webdriver
+from bs4 import BeautifulSoup
+import requests
+
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.keys import Keys
+import time
+
+import sys
+
+class WalmartScraper:
+    zipSet = False
+    def __init__(self):
+        chrome_options = Options()
+        chrome_options.add_argument('--headless')
+        try:
+            self.driver = webdriver.Firefox(executable_path=r'.\geckodriver.exe')

```

Comment:
Can we get it so that this script takes in an argument of either `chrome` or `firefox` that lets it choose which browser to use?